docs/src/developer/getting_started/index.md

```
@@ -0,0 +1,14 @@
+# Getting Started für Entwickler
+
+![wrench](/wrench.png)
+
+to be done...
+
+---
+
+Todos:
+- Set up IDE (VSCode)
```

Markdownlint (0.35.0)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)